tests/data/utils/test_compute_shape_offset.py

```
@@ -0,0 +1,49 @@
+
+from monai.data.utils import compute_shape_offset
+
+
+class TestComputeShapeOffset(unittest.TestCase):
+
+    def setUp(self):
+        self.affine = np.eye(4)
+
+    def test_numpy_array_input(self):
+        shape = np.array([64, 64, 64])
+        out_shape, offset = compute_shape_offset(shape, self.affine, self.affine)
```

Ruff (0.15.9)

[warning] Unpacked variable `offset` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)